pages/your_domain/TEMPLATE_your_domain.md

```diff
@@ -4,33 +4,32 @@ keywords: optional
 search: exclude
 ---
 
-## Overview
+## Introduction
 
-A brief overview of the domain as a whole and the particular challenges it raises.
+Domain pages should detail the particular data management challenges of the domain, typically by complementing and extending one or more existing Problem pages.
```

Comment:
I suggest to put the explanation of what is expected in this template between special symbols.
<!-- domain page should... -->
In this way it will be visible in markdown but not display it in the website.

Reply:
Done